### PR TITLE
added javadoc to announcements dto

### PR DESCRIPTION
**Pull request by jackielincroft**

### CODE REVIEW

File: api/src/main/java/com/codeforcommunity/dto/announcements/Announcement.java [hunk 1]
```diff
@@ -3,6 +3,10 @@
 import com.fasterxml.jackson.annotation.JsonFormat;
 import java.sql.Timestamp;
 
+/**
+ * Represents the data about an announcement, including announcement's id, title, description,
+ * id of relevant event, and timestamp of when the announcement was created.
+ */
 public class Announcement {
```
Comment (anirudh723) [inline api/src/main/java/com/codeforcommunity/dto/announcements/Announcement.java:10]: Looks good!

File: api/src/main/java/com/codeforcommunity/dto/announcements/PostAnnouncementResponse.java [hunk 2]
```diff
@@ -1,15 +1,27 @@
 package com.codeforcommunity.dto.announcements;
 
+/**
+ * Represents an object containing the information for a response to a post announcement,
+ * which is just the announcement.
+ */
```
Comment (anirudh723) [inline api/src/main/java/com/codeforcommunity/dto/announcements/PostAnnouncementResponse.java:6]: 👍

**Comment by anirudh723** [comment]
Looks like the build failed because of the formatting violations. Run `mvn spotless:apply` and commit again

**Review by anirudh723 — COMMENTED** [review]
Looks like the build failed because of the formatting violations. Run `mvn spotless:apply` and commit again

**Comment by jackielincroft** [comment]
> Looks like the build failed because of the formatting violations. Run `mvn spotless:apply` and commit again

Awesome thank you for the note!!

**Review by anirudh723 — APPROVED** [review]
LGTM. Glad you were able to fix the issue fast!

**Pull request merged**